keyboards/ocean/gin_v2/readme.md

```diff
@@ -0,0 +1,24 @@
+# Gin V2
+
+![Gin V2](https://i.imgur.com/CQiImvu.jpeg)
```

Comment:
```suggestion
![Gin V2](https://i.imgur.com/CQiImvuh.jpeg)
```

Reply:
https://i.imgur.com/CQiImvu.jpeg it is already correct